postgres/src/test/scala/zio/sql/postgresql/PostgresModuleTest.scala

```diff
@@ -0,0 +1,130 @@
+package zio.sql.postgresql
+
+import java.time.LocalDate
+import java.util.UUID
+
+import zio.sql.postgresql.ShopSchema
+import zio.sql.postgresql.PostgresModule
+import zio.test._
+import zio.test.Assertion._
+
+object PostgresModuleTest
+    extends DefaultRunnableSpec
+    with PostgresIntegrationTestBase
+    with PostgresModule
+    with ShopSchema {
+
+  import this.Customers._
+  import this.Orders._
+
+  val spec = suite("Postgres module")(
+    testM("Can select from single table") {
+      case class Customer(id: UUID, fname: String, lname: String, dateOfBirth: LocalDate)
+      
+      val query = select(customerId ++ fName ++ lName ++ dob) from customers
+
+      val expected =
+        Seq(
+          Customer(
+            UUID.fromString("60b01fc9-c902-4468-8d49-3c0f989def37"),
+            "Ronald",
+            "Russell",
+            LocalDate.parse("1983-01-05")
+          ),
+          Customer(
+            UUID.fromString("f76c9ace-be07-4bf3-bd4c-4a9c62882e64"),
+            "Terrence",
+            "Noel",
+            LocalDate.parse("1999-11-02")
+          ),
+          Customer(
+            UUID.fromString("784426a5-b90a-4759-afbb-571b7a0ba35e"),
+            "Mila",
+            "Paterso",
+            LocalDate.parse("1990-11-16")
+          ),
+          Customer(
+            UUID.fromString("df8215a2-d5fd-4c6c-9984-801a1b3a2a0b"),
+            "Alana",
+            "Murray",
+            LocalDate.parse("1995-11-12")
+          ),
+          Customer(
+            UUID.fromString("636ae137-5b1a-4c8c-b11f-c47c624d9cdc"),
+            "Jose",
+            "Wiggins",
+            LocalDate.parse("1987-03-23")
+          )
+        )
+
+      val testResult = new ExecuteBuilder(query)
+        .to[UUID, String, String, LocalDate, Customer] { case row =>
```

Comment:
not for fixing with this PR but I think we need to come up with a better way of extracting test results. We shouldn't have to specify the types as this is encoded in the type tags... just need to figure out how to get them!